src/test/java/org/opensearch/sdk/TestNetty4Transport.java

```diff
@@ -12,6 +12,7 @@
 package org.opensearch.sdk;
 
 import java.io.IOException;
+
```

Comment:
Is this line added by Spotless?

Reply:
no, I will delete this line in next commit.